.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Description of work:
+
+- [ ] Source added/refactored?
+- [ ] Added unit tests?
+- [ ] Added integration tests?
+
+### To Test by Reviewer:
+
+<!-- Instructions for testing. -->
+
+The reviewer should check for the discussed software practices
+
+- [ ] all internal functions should have an underbar, as is python standard
+- [ ] need more comments in meat of code explaining intent
+- [ ] clearly named variables (better to be verbose in variable names)
+- [ ] does the code follow the other [coding standards](https://scse.ornl.gov/drtsans/developer/index.html)?
```

Comment:
I got a "down for maintenance" page when I clicked this link. Is it down temporarily or permanently? (I want to read it :smile_cat:)

Reply:
oops, I didn't even now I inserted this. I blinded did a copy/paste from `drtsans` pull request template. I'll get rid of it.